docs/resources/ddm_account.md

```diff
@@ -0,0 +1,95 @@
+---
+subcategory: "Distributed Database Middleware (DDM)"
+---
+
+# huaweicloud_ddm_account
+
+Manages a DDM account resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "ddm_instance_id" {}
+variable "name" {}
+variable "password" {}
+variable "schema_name" {}
+
+resource "huaweicloud_ddm_account" "test"{
+  instance_id = var.ddm_instance_id
+  name        = var.name
+  password    = var.password
+
+  permissions = [
+    "CREATE",
+    "SELECT"
+  ]
+
+  schemas {
+   name = var.schema_name
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of a DDM instance.
+
```

Comment:
Please remove the blank line and check the below.

Reply:
done